src/Akeneo/Pim/Enrichment/Component/Product/Commands/GroupProductsCommand.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+
+namespace Akeneo\Pim\Enrichment\Component\Product\Commands;
```

Comment:
```suggestion
namespace Akeneo\Pim\Enrichment\Component\Product\Command;
```

Usually we don't pluralize the namespace

Reply:
done!